apps/notifications/tests/utils.py

```diff
@@ -0,0 +1,38 @@
+import base64
```

Comment:
Is this file in use? I failed to find a reference to it in the PR

Reply:
Huh, you are correct. This is a leftover from earlier versions of this feature. I think it allows us to test actual notification delivery in our tests, which could be very useful

Reply:
Can I suggest we either use it or remove it then? If it is not being used, I am afraid it will end up as clutter as it's existence and purpose might not be clear for future workers on this

Reply:
I created a test with a use for it :smile: